test/internal/util/util.go

```diff
@@ -0,0 +1,20 @@
+package util
+
+import (
+	"strings"
+	"unicode"
+)
+
+// CleanName returns a string appropriate for uVM, container, or file names.
+//
+// Based on [testing.TB.TempDir].
+func CleanName(n string) string {
```

Comment:
Why is this needed?

Reply:
So `"testing".TB.TempDir` has code to turn a test name into a string appropriate for use as temp folder name.
I needed something similar to use when creating folders for the images (`extractLayers`), and also to set the names for the LCOW and WCOW uVMs based